### PR TITLE
test: `servicesnames` field in `getpeerinfo` and `getnetworkinfo`

### LINKED ISSUE #16844: test: Add smoke test for getpeerinfo['servicesnames']
https://github.com/bitcoin/bitcoin/pull/16787#issuecomment-529801457

### PR DESCRIPTION
**Pull request by darosior**
As per https://github.com/bitcoin/bitcoin/pull/16787#issuecomment-529801457, fixes #16844.

This adds a test for both commands in the first commit and renames the test for `getnetworkinfo` in the second commit.

### CODE REVIEW

**Comment by darosior** [comment]
Could someone restart the [stopped job (583339836)](https://travis-ci.org/bitcoin/bitcoin/jobs/583339836) please ?

File: test/functional/rpc_net.py [hunk 1]
```diff
@@ -104,6 +125,18 @@ def _test_getpeerinfo(self):
         assert_equal(peer_info[1][0]['addrbind'], peer_info[0][0]['addr'])
         assert_equal(peer_info[0][0]['minfeefilter'], Decimal("0.00000500"))
         assert_equal(peer_info[1][0]['minfeefilter'], Decimal("0.00001000"))
+        # check the `servicesnames` field
+        for info in peer_info:
+            if int(info[0]["services"]) & NODE_NETWORK:
```
Comment (MarcoFalke) [inline test/functional/rpc_net.py]: Could move this into a function for less duplication?

```py
assert_network_service_names(services=int(info[0]["services"]), names=info[0]["servicesnames"])

**Review by maflcko — APPROVED** [review]
ACK

**Review by jamesob — COMMENTED** [review]
Concept ACK, thanks for writing tests.

**Comment by darosior** [comment]
Rebased to regroup into a function as suggested by @MarcoFalke.

File: test/functional/rpc_net.py (hunk 1 above)
Comment (darosior) [reply test/functional/rpc_net.py]: Done

**Comment by laanwj** [comment]
ACK 1d524c62ea679aa89770f9fdcd72b84f013639cb

**Pull request merged**